pre-docs/inventory_file.md

```diff
@@ -195,10 +195,28 @@ nodes:
           run-as: root
 ```
 
--   **[Generating inventory files](inventory_file_generating.md)**  
+## Remote Targets
+
+Configure a remote target. When using the remote transport the protocol of the
+node name does not have to map to the transport if you set the transport config
+option. This is useful if the target is an http API as in the following example.
+
+```yaml
+nodes:
+  - host1.example.com
+  - name: https://user1:secret@remote.example.com
+    config:
+      transport: remote
+      remote:
+        run-on: host1.example.com
```

Comment:
This defaults to using `host1` over SSH? Would be useful to expand the example somehow to clarify how you get config for host1.example.com.